cdap-ui/app/features/tracker/services/my-lineage-service.js

```diff
@@ -266,16 +266,30 @@ class myLineageService {
   parseProgramType(programType) {
     switch (programType) {
       case 'Flow':
+      case 'flow':
+      case 'Flows':
+      case 'flows':
         return 'flows';
       case 'Mapreduce':
+      case 'mapreduce':
         return 'mapreduce';
       case 'Spark':
+      case 'spark':
         return 'spark';
       case 'Worker':
+      case 'worker':
```

Comment:
lineage will return Uppercase (Flow), while tracker top entity will return lowercase (flow). The UI sometimes need to display in plural format as well.